### DISCUSSION TITLE
Add Support for File types in Open API Connector

### DISCUSSION THREAD

**Discussion by neel015** [microsoft/semantic-kernel#8834] [Ideas]
Hello SK team!

We have been using the package `Microsoft.SemanticKernel.Plugins.OpenApi` , it works well with API requests accepting JSON.
However, we have an API which accepts file, for that it is not working. 
While taking a look at the [repo](https://github.com/microsoft/semantic-kernel/blob/9529652d5d36903170660f02228744220e72f466/dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs#L68)  we can see that only `application/json` and `text/plain` are supported.

It would be nice to support file types in API requests.

**Comment by evchaki** [comment]
@neel015 - we will make this an issue to make sure we get this in for you.
